JunkyardLoad/JunkyardLoad.cs

```diff
@@ -107,6 +113,12 @@ public static async Task JunkyardNet8LinuxDevProfilerAll([TimerTrigger(LoadTestI
             await JunkyardDump("dd-dotnet-linux-latest-build-profiler-all", log: log, service: "net8-linux-latest-build-profiler-all");
         }
 
+        [FunctionName("dd-dotnet-security-aspnetcore")]
```

Comment:
You added a constant for this 😄 

```suggestion
        [FunctionName(SecurityAppUrlPrefix)]
```